app/grants/templates/grants/hall_of_fame.html

```diff
@@ -0,0 +1,208 @@
+                        </div>
+                    </div>
+                    <div class="d-block d-sm-none pt-4"></div>
+                    <div class="d-none d-sm-block pt-3"></div>
+                    <div class="text-center">
+                        <a class="btn btn-primary btn-lg" href="{%url 'grants:new_matching_partner' %}">
+                            Join the Funder's League
+                        </a>
+                    </div>
+                    <div class="d-block d-sm-none pt-5"></div>
+                    <div class="d-none d-sm-block pt-4"></div>
+                    <div class="d-none d-sm-block pt-4"></div>
+                </div>
+            </div>
+
+            <div class="container-fluid px-0">
+                <div id="partner_section" class="container content text-left">
+                    <div class="d-block d-sm-none pt-4"></div>
+                    <div class="d-none d-sm-block pt-5"></div>
+                    <h2 class="text-grey-500">{%trans "Top Matching Partners" %}</h2>
+                    <div class="pt-3"></div>
+
+                    <picture>
+                        <source media="(max-width: 320px)" srcset="{{ top_matching_partners_mobile_url }}">
+                        <source media="(min-width: 321px)" srcset="{{ top_matching_partners_url }}">
+                        <img class="w-100" src="{{ top_matching_partners_url }}" alt="Top matching partners">
+                    </picture>
+                    <div class="pt-5"></div>
+                </div>
+            </div>
+
+            <div class="container-fluid pb-5 hof-bg-violet-500">
+                <div class="container content text-left">
+                    <div class="d-block d-sm-none pt-4"></div>
+                    <div class="d-none d-sm-block pt-5"></div>
+                    <h2 class="text-white">{%trans "Top Individual Grant Donors" %}</h2>
+                    <div class="pt-3"></div>
+                    <picture>
+                        <source media="(max-width: 320px)" srcset="{{ top_individual_donors_mobile_url }}">
+                        <source media="(min-width: 321px)" srcset="{{ top_individual_donors_url }}">
+                        <img class="w-100" src="{{ top_individual_donors_url }}" alt="Top individual donors">
+                    </picture>
+                </div>
+            </div>
+
+            <div class="d-block d-sm-none pt-4"></div>
+            <div class="d-none d-sm-block pt-5"></div>
+
+            <div class="container text-left pb-5">
+                <h2 class="text-grey-500">{%trans "Graduated Grantees" %}</h2>
+                <div class="mt-3 text-grey-400">{{ graduated_grantees_description }}</div>
+
+                <div class="d-block d-sm-none pt-4"></div>
+                <div class="d-none d-sm-block pt-5"></div>
+
+                <div class="container">
+                    {% for g in graduated_grantees %}
+                    {% if not forloop.first %}
```

Comment:
let's have this indented by 1 for sake of readability

Reply:
This is done.